linode/manager
# M3-1834 Character decoding on blog feed

## Purpose
This PR fixes an issue where certain characters coming in from the blog feed weren't being decoded properly.

We previously implemented a temporary fix for the ellipsis character only (#3941).

This solution uses the "he" library for decoding. ["He"](https://www.npmjs.com/package/he) is an extremely well tested library for doing this sort of thing, and has ~4 million weekly NPM downloads.

## To test:
Navigate to the main dashboard. At the time of this PR, the top story is titled "Now Available: Linode Terraform Provider". On production, the description contains this sentence:

`Terraform is a popular open source infrastructure as code tool that&#8217;s quickly becoming the industry standard for infrastructure automation.`

With this PR, the sentence should read: 

`Terraform is a popular open source infrastructure as code tool that’s quickly becoming the industry standard for infrastructure automation.`

## Changes: 
- Our previous solution did a `replace()` on ` [&#8230;]`, but in this PR, only the HTML code is replaced (and not the brackets). This means that the trailing ellipsis includes the bracket now:
![screen shot 2018-11-26 at 10 29 04 am](https://user-images.githubusercontent.com/16911484/49023866-21c02980-f166-11e8-828b-a27355b35f39.png)
